java-client/src/test/java/org/opensearch/client/opensearch/ingest/TextEmbeddingProcessorTest.java

```diff
@@ -25,30 +25,69 @@ private static TextEmbeddingProcessor.Builder baseTextEmbeddingProcessor() {
     }
 
     @Test
-    public void testJsonRoundtripWithDescription() {
+    public void testJsonRoundtripWithDescriptionAndBatchSize() {
         Processor processor = new Processor.Builder().textEmbedding(
-            baseTextEmbeddingProcessor().description("processor-description").build()
+            baseTextEmbeddingProcessor().description("processor-description").batchSize(1).build()
         ).build();
         String json =
-            "{\"text_embedding\":{\"tag\":\"some-tag\",\"model_id\":\"modelId\",\"field_map\":{\"input_field\":\"vector_field\"},\"description\":\"processor-description\"}}";
+            "{\"text_embedding\":{\"tag\":\"some-tag\",\"model_id\":\"modelId\",\"field_map\":{\"input_field\":\"vector_field\"},\"description\":\"processor-description\",\"batch_size\":1}}";
         TextEmbeddingProcessor deserialized = checkJsonRoundtrip(processor, json).textEmbedding();
 
         assertEquals("modelId", deserialized.modelId());
         assertEquals(baseFieldMap, deserialized.fieldMap());
         assertEquals("processor-description", deserialized.description());
         assertEquals("some-tag", deserialized.tag());
+        assert deserialized.batchSize() != null;
```

Comment:
This should be `assertNotNull(deserialized.batchSize())`

Reply:
As you suggested, it would be better to change this line using [assertNotNull](https://junit.org/junit4/javadoc/4.13/src-html/org/junit/Assert.html#line.712) for readability and test convention.